scripts/check_doc_ipfs_hashes.py

```diff
@@ -168,32 +188,24 @@ def get_hash_by_package_line(
             return None
 
 
-def update_test_files(old_to_new_hashes: Dict[str, str]) -> None:
```

Comment:
Is this not needed?

Reply:
No, it was changed in open-autonomy a couple of weeks ago. I've now ported those changes to here.